cluster/addons/calico-policy-controller/typha-horizontal-autoscaler-role.yaml

```diff
@@ -0,0 +1,16 @@
+apiVersion: rbac.authorization.k8s.io/v1beta1
+kind: Role
+metadata:
+  name: typha-horizontal-autoscaler
+  namespace: kube-system
+  labels:
+    kubernetes.io/cluster-service: "true"
+    addonmanager.kubernetes.io/mode: Reconcile
+rules:
+  - apiGroups: [""]
+    resources: ["configmaps"]
+    verbs: ["get"]
+  - apiGroups: ["extensions"]
+    resources: ["deployments/scale"]
+    verbs: ["get"]
```

Comment:
This autoscaler will also need to update the deployments/scale resource - [see here](https://github.com/kubernetes-incubator/cluster-proportional-autoscaler/blob/master/examples/RBAC/RBAC-configs.yaml#L32)